NuGet.Docs/docs/Get-Started/Install-NuGet.md

```diff
@@ -1,13 +1,112 @@
-#Install NuGet
+#NuGet Tools
+
+Whether consuming or creating NuGet packages you can use the following tools to meet your needs:
+
+##NuGet.exe Command-line Tool
+This utility can be used to create, publish, and download packages. 
+
+How do I get NuGet.exe?
+<ol>
+<li>1. Download the latest version from here and save the file to a path of your choice.</li>
+<li>2. Add this path to the PATH Environment Variable.</li>
+</ol>
+
+<div class="block-callout-warning">
+	<strong>NuGet 2.x users</strong><br>
+    Because there are a few breaking changes introduced in NuGet 3.2 https://nuget.org/nuget.exe is pointing to the NuGet 2.x latest release, to prevent CI systems from auto upgrading and potentially breaking.
+</div>
+
+####Try out some scenarios
+*  [Creating a package](https://docs.nuget.org/Create/Creating-and-Publishing-a-Package#creating-a-package)
+*  [Publishing a Package](https://docs.nuget.org/Create/Creating-and-Publishing-a-Package#publishing-using-nuget-command-line)
+
+####References
+* [NuGet.exe commands](https://docs.nuget.org/consume/command-line-reference)
+
+##NuGet Package Manager extension in Visual Studio
+Starting with Visual Studio 2012, NuGet is included in every edition (except Team Foundation Server).. This extension includes the package manager UI and the package manager console.
+
+If your copy of Visual Studio does not already have the NuGet Package Manager extension, you can install it using the Extension Manager.<br>
+<ol>
+<li>1. In Visual Studio, click Tools and then Extension and Updates.</li>
+<li>2. Navigating to Online, search for "NuGet Package Manager for Visual Studio" and click Download.</li>
+<li>3. In the Installer dialog box, click Install.</li>
+<li>4. When installation is complete, close and re-open Visual Studio.</li>
+</ol>
+
+
+###Package Manager UI
+The package manager UI allows you to find, install, remove, and update NuGet packages using Manage NuGet Packages dialog box.
+
+####Try out some scenarios
+* [Finding and installing a package](http://docs.nuget.org/Consume/Package-Manager-Dialog#managing-nuget-packages-using-the-dialog)</li>
+* [Removing a package](http://docs.nuget.org/Consume/Package-Manager-Dialog#removing-a-package)</li>
+* [Updating a Package](http://docs.nuget.org/Consume/Package-Manager-Dialog#updating-a-package)</li>
+
+
+###Package Manager Console
+The package manager console allows you to find, install, remove, and update NuGet packages using PowerShell commands. Some packages and tools during their install (like EntityFramework.) which can be accessed through the package manager console.
+
+<div class="block-callout-warning">
+	<strong>Note</strong><br>
+    Package Manager Console commands do not work outside of visual studio since it is dependent on VS objects like DTE to work correctly.
+</div>
+
+
+####Try out some scenarios
+* [Finding and installing a package](https://docs.nuget.org/Consume/Package-Manager-Console#finding-a-package)</li>
```

Comment:
Removing the "Finding and installing a package". The other two examples still remain.